Mathlib/Tactic/SimpRw.lean

```diff
@@ -59,6 +60,6 @@ elab s:"simp_rw " rws:rwRuleSeq g:(location)? : tactic => do
     evalTactic (← match term with
     | `(term| $e:term) =>
       if symm then
-        `(tactic| simp%$e only [←$e:term] $g ?)
+        `(tactic| fail_if_no_progress simp%$e only [←$e:term] $g ?)
```

Comment:
Is it possible to make this just a warning on the unused lemma, as I think happens in `simp`?

Reply:
Yes, a warning would be better.

In any case this PR isn't viable, it will require https://github.com/leanprover-community/mathlib4/pull/3757 first.